recipes/taproom/meta.yaml

```
@@ -0,0 +1,42 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+{% set name = "taproom" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slochower/host-guest-benchmarks/archive/v{{ version }}.tar.gz
+  sha256: ab30eb20fb33a4387db1f9b5824fb78d395e0788c97693f7fcf6bacd382c854a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pyyaml
```

Do you need to add ``pydantic`` as a dependency?
```suggestion
    - pyyaml
    - pydantic
```

---

Technically, yes, though nothing uses that stub currently. But I added `pydantic` because that seemed to be the right thing to do. Done in e032d79. Ready for-rereview.